samples/client/echo_api/python-pydantic-v1/test/test_manual.py

```diff
@@ -145,6 +145,11 @@ def testAuthHttpBasic(self):
         self.assertTrue("Authorization" in e.headers)
         self.assertEqual(e.headers["Authorization"], "Basic dGVzdF91c2VybmFtZTp0ZXN0X3Bhc3N3b3Jk")
 
+    def testParametersToUrlQuery(self):
```

Comment:
@mhaley-tignis for consistency, can you rename this method and those in the other tests with the same name?

I think `test_parameters_to_url_query_boolean_value` is a good name and clearly explain the test :)

Reply:
I can, see f701d7daa50f1.  Most of the other tests in this file follow camel casing, want me to change them over too?

Reply:
No, you can keep it this way I think.